lib/pure/unicode.nim

```diff
@@ -34,15 +34,15 @@ proc `==`*(a, b: Rune): bool = return int(a) == int(b)
 
 template ones(n: untyped): untyped = ((1 shl n)-1)
 
-proc runeLen*(r: Rune): int {.noSideEffect.} =
+proc size*(r: Rune): int {.noSideEffect.} =
```

Comment:
``size`` is a worse name than ``runeLen``. :-( 

https://nim-lang.org/docs/apis.html

"size should refer to a byte size"

Reply:
@Araq @dom96 I am reverting this back to `runeLen`. At least then it will be somewhat consistent with its sibling proc `runeLenAt`.